pkgs/development/compilers/cudatoolkit/saxpy/default.nix

```diff
@@ -0,0 +1,50 @@
+{ backendStdenv
+, cmake
+, cuda_cccl
+, cuda_cudart
+, cudaFlags
+, cuda_nvcc
+, lib
+, libcublas
+, setupCudaPathsHook
+, stdenv
+, autoAddOpenGLRunpathHook
+}:
+
+backendStdenv.mkDerivation {
+  pname = "saxpy";
+  version = "unstable-2023-07-11";
+
+  src = ./.;
+
+  buildInputs = [
+    libcublas
+    cuda_cudart
+    cuda_cccl
+  ];
+  nativeBuildInputs = [
+    cmake
+
+    # NOTE: this needs to be pkgs.buildPackages.cudaPackages_XX_Y.cuda_nvcc for
+    # cross-compilation to work. This should work automatically once we move to
+    # spliced scopes. Delete this comment once that happens
```

Comment:
Nit: Would you mind making an issue for that and putting a link to it here? I feel like that gives a better way to track issues than inline comments.

Reply:
Sure, I'll make one and keep both: I rely on comments when navigating git blame, and issues when navigating github